tool/net/redbean.c

```diff
@@ -1062,6 +1062,35 @@ static void Daemonize(void) {
   ChangeUser();
 }
 
+static int LuaCallWithTrace(lua_State *L, int nargs, int nres) {
+  int nresults, status;
+  // create a coroutine to retrieve traceback on failure
+  lua_State *co = lua_newthread(L);
```

Comment:
Now I'm so happy I asked you. There's a level of Lua wizardry happening here I would not have easily been able to figure out on my own. Good show.